editoast/src/infra.rs

```diff
@@ -64,6 +73,24 @@ impl ApiError for InfraApiError {
 }
 
 impl Infra {
+    #[cfg(test)]
+    pub fn find_by_name(
+        conn: &PgConnection,
+        infra_name: String,
+    ) -> Result<Infra, Box<dyn ApiError>> {
+        let infra_name_error_msg = infra_name.clone();
+        match osrd_infra_infra
+            .filter(name.eq(infra_name))
+            .get_result(conn)
+        {
+            Ok(infra) => Ok(infra),
+            Err(DieselError::NotFound) => Err(Box::new(InfraApiError::NotFoundByName(
+                infra_name_error_msg,
+            ))),
+            Err(e) => Err(Box::new(InfraApiError::DieselError(e))),
+        }
+    }
```

Comment:
It can be used if needed. But because of compilation warning I annotated it with #[cfg(test)]. Should I put it elsewhere? I have no other solution idea.

Reply:
I'll move it in test. Because it could be used in futur, I thought it would be a good place.

Reply:
I moved it in tests